drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/editor/RuleModellerActionSelectorPopup.java

```diff
@@ -279,8 +280,8 @@ public void execute() {
     }
 
     //Add Modifies
-    private void addModifies() {
-        List<String> vars = model.getAllLHSVariables();
```

Comment:
@jomarko I think you are correct. I thought it did not make sense to be able to "modify" bound fields; but I've just tried with vanilla DRL and it seems quite valid. I also checked the Guided Rule Editor with 6.5.x and "modifying" a bound field works fine.